backend/t4cclient/routes/repositories/__init__.py

```diff
@@ -82,6 +82,7 @@ def create_repository(
     token=Depends(JWTBearer()),
 ):
     verify_admin(token, db)
+    connection.add_user_to_repository(body.name, get_username(token), isAdmin=True)
```

Comment:
What is your intention here? Beside the fact, that the repository doesn't even exist at this point (because it is only created in the next step), we also don't need to create the user here as it would use a random password.